neurots/generate/orientations.py

```diff
@@ -59,7 +59,7 @@ def __init__(self, soma, parameters, distributions, context, rng):
         self._soma = soma
         self._parameters = parameters
         self._distributions = distributions
-        self._context = context
+        self._context = context if context is not None else {}
```

Comment:
Why is this different from the correction of context in grower?

self.context = self._process_context(context)

Reply:
Indeed, if NeuronGrower has already processed it can it be None at this point?

Reply:
yes, right, I put this to be sure it does not crash if used alone at all (like in the tests), as the context has to be a dict, as we use .get(). But default to {} in the parameter is dangerous.

Reply:
@adrien-berchet maybe there is a better way to do this?

Reply:
Yep, makes sense.  I agree with the default {} not being a good solution. I don't think this is critical, if you have a better alternative ok otherwise it's ok as it is.

Reply:
Yeah I think it's ok like this